src/CoreBundle/EventListener/AddGlobalHeadIncludesListener.php

```diff
@@ -38,12 +38,16 @@ public function __construct(\TPkgViewRendererSnippetDirectoryInterface $viewRend
      */
     public function onGlobalHtmlHeaderInclude(HtmlIncludeEventInterface $event)
     {
+        $event->addData([
+            '<script src="'.TGlobal::GetStaticURLToWebLib('/javascript/cms.js').'" type="text/javascript"></script>',
+        ]);
```

Comment:
this would also inject the cms.js into the frontend, would it not? I think we should avoid that, unless it is also used in the frontend.

Reply:
True, checking that now.